satpy/tests/reader_tests/test_agri_l1.py

```diff
@@ -38,12 +37,20 @@
                           'GEO': 'solar_azimuth_angle'
                           }
 
-AREA_EXTENTS_BY_RESOLUTION = {
-    500:  (-5495771.007913081, 5495271.006001793, -5493771.000267932, 5495771.007913081),
-    1000: (-5495521.074086424, 5494521.070251633, -5491521.058747265, 5495521.074086424),
-    2000: (-5495021.206414789, 5493021.198696349, -5487021.175541028, 5495021.206414789),
-    4000: (-5494021.20255557, 5490021.187118688, -5478021.140808046, 5494021.20255557)
-}
+RESOLUTION_LIST = [500, 1000, 2000, 4000]
+
+AREA_EXTENTS_BY_RESOLUTION = {'FY4A': {
+    500: (-5496021.008869, 5495021.005046, -5493520.999312, 5496021.008869),
+    1000: (-5496021.076004, 5494021.068334, -5491021.05683, 5496021.076004),
+    2000: (-5496021.210274, 5492021.194837, -5486021.171682, 5496021.210274),
+    4000: (-5496021.210274, 5488021.1794, -5476021.13309,  5496021.210274)
+},
+    'FY4B': {
+    500: (-5496021.008869, 5495021.005046, -5493520.999312, 5496021.008869),
+    1000: (-5496021.076004, 5494021.068334, -5491021.05683, 5496021.076004),
+    2000: (-5496021.210274, 5492021.194837, -5486021.171682, 5496021.210274),
+    4000: (-5496021.210274, 5488021.1794, -5476021.13309, 5496021.210274)
```

Comment:
I can't see any difference between fy4a and fy4b extents, should they be factorised?